src/templates/fr/2019/methodology.html

```diff
@@ -0,0 +1,516 @@
+      </p>
+
+      <p>
+        Pour contourner certaines des limites inhérentes à un jeu de données issus de tests synthétiques, le Web Almanac utilise également le  <a href="#chrome-ux-report">Chrome UX Report</a> afin de récupérer des indicateurs relatifs à l’expérience utilisateur, notamment dans le domaine des performances Web.
+      </p>
+
+      <p>
+        Certaines informations restent totalement inaccessibles. Par exemple,  nous ne sommes pas toujours en mesure de détecter les outils utilisés pour construire un site Web. Si un site Web est construit à l’aide de l’application create-react, nous pourrons dire qu’il utilise le framework React, mais pas nécessairement que cet outil de construction particulier a été utilisé. À moins que ces outils ne laissent des empreintes digitales détectables dans le code du site Web, nous ne pouvons pas mesurer leur utilisation.
+      </p>
+
+      <p>
+        Pour d’autres paramètres, la mesure n’est pas impossible mais elle est difficile à mesurer ou peu fiable. Par exemple, certains aspects de la conception Web sont intrinsèquement visuels et peuvent être difficiles à quantifier, comme le fait qu’une page ait une pop-in intrusive.
+      </p>
+    </section>
+
+    <section>
+      <h2 id="tools">Outils</h2>
+
+      <p>
+        Le Web Almanac est rendu possible grâce aux outils open source suivants.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="webpagetest">WebPageTest</h3>
+
+      <p>
+        <a href="https://www.webpagetest.org/">WebPageTest</a> est un outil de test de performance Web incontournable et l'épine dorsale de HTTP Archive. Nous utilisons une <a   href="https://github.com/WPO-Foundation/webpagetest-docs/blob/master/user/Private%20Instances/README.md"   >instance privée</a > de WebPageTest avec des agents de test privés, c’est-à-dire de vrais navigateurs testant chaque page web. Les sites pour ordinateurs de bureau et périphériques mobiles sont testés avec différentes configurations&nbsp;:
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+              <th>Config</th>
+              <th>Bureau</th>
+              <th>Mobile</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Matériel</td>
+            <td>Linux VM</td>
+            <td>Moto G4 Émulé</td>
+          </tr>
+
+          <tr>
+            <td>User Agent</td>
+            <td>
+              Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML,
+              like Gecko) Chrome/75.0.3770.100 Safari/537.36
+              PTST/190704.170731
+            </td>
+            <td>
+              Mozilla/5.0 (Linux; Android 6.0.1; Moto G (4)
+              Build/MPJ24.139-64) AppleWebKit/537.36 (KHTML, like Gecko)
+              Chrome/65.0.3325.146 Mobile Safari/537.36 PTST/190628.140653
```

Comment:
Can we merge these lines similar to English version? After that I'm happy to approve.

Reply:
Gladly